packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotController.java

```diff
@@ -838,15 +838,18 @@ private ListenableFuture<MediaPlayer> loadCameraSound() {
     }
 
     private void playCameraSound() {
-        mCameraSound.addListener(() -> {
-            try {
-                MediaPlayer player = mCameraSound.get();
-                if (player != null) {
-                    player.start();
+        if (Settings.System.getInt(mContext.getContentResolver(),
```

Comment:
Better to return early instead, it'll likely simplify porting.

Reply:
Made it return early if the setting is not true (enabled)